manifests/01-wireguard-kmmo.yaml

```diff
@@ -31,15 +31,15 @@ spec:
     - configMap:
         name: wireguard-kmod-helpers
     dockerfile: |
-      FROM quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:0661d0560654e7e2d1761e883ffdd6c482c8c8f37e60608bb59c44fa81a3f0bb
+      FROM quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:397bdb1550d3d620376760449a14147266e87a5486a85e1381b10052daa0b392
 
       ARG ARTIFACTS_LOCATION
       ARG WIREGUARD_ARCHIVE_NAME
       ARG WIREGUARD_ARCHIVE_SHA256
 
 
       WORKDIR /
-      RUN cat /etc/systemd/system/kmod*
+      #RUN cat /etc/systemd/system/kmod*
```

Comment:
this was a debugging line.. can definitely delete this instead of commenting

Reply:
done